src/Generator.php

```diff
@@ -50,26 +50,44 @@ public function __construct(Config $config, Version $version = null)
     }
 
     /**
-     * Compile API documentation into a parseable array.
+     * Compile API documentation into a parseable collection.
      *
      * @return array
      */
     public function generate()
     {
-        $this->compileResources();
-        $this->compileRepresentations();
+        // Generate resources.
+        $resources = $this->compileResources($this->parseResources());
+        foreach ($resources as $version => $groups) {
+            // Alphabetize the versioned resource groups!
+            uksort($resources[$version], function ($a, $b) {
```

Comment:
why `uksort` instead of `ksort`? Isn't the supplied function the same as default (except you are never returning 0)?